ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/inmemory/InMemoryLedger.scala

```diff
@@ -336,13 +336,15 @@ class InMemoryLedger(
       }
     }
 
-  override def lookupLedgerConfiguration(): Future[Option[Configuration]] =
-    Future.successful(this.synchronized { ledgerConfiguration })
+  override def lookupLedgerConfiguration(): Future[Option[(Long, Configuration)]] =
+    Future.successful(this.synchronized {
+      ledgerConfiguration.map(config => ledgerEnd -> config)
+    })
 
   override def configurationEntries(
-      offset: Option[Long]): Source[(Long, ConfigurationEntry), NotUsed] =
+      startExclusive: Option[Long]): Source[(Long, ConfigurationEntry), NotUsed] =
```

Comment:
Please make this beginInclusive, which is the semantics we use for other offsets as well.

Reply:
I don't know how to make it work when we remove internal offsets and use external "opaque" offsets. We cannot return "the next offset" from "lookupConfiguration" as we don't know what it is.